scripts/utils/generate_package_wiki.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import os
+import pathlib
+import sys
+import xml.etree.ElementTree as ET
+from collections import defaultdict
+
+packagesByCategory = defaultdict(str)
+
+
+def sort_write_wiki_content(file_path):
+    """
+    iterates over each category from the dictionary packagesByCategory and
+    writes the Markdown formatted string that contains
+    - header with the category name
+    - table with two columns: Package and Description of each of the packages"""
+    wikiContent = ""
+    for category in packagesByCategory.keys():
+        wikiContent += "## " + category + "\n\n"
+        wikiContent += "| Package | Description |\n"
+        wikiContent += "|---|---|\n"
+        wikiContent += packagesByCategory[category] + "\n\n"
+    with open(file_path, "w", encoding="utf-8") as f:
+        f.write(wikiContent)
+
+
+# Helper function to find elements
+def find_element(parent, tag_name):
+    return parent.find(f"{{*}}{tag_name}")
```

Comment:
Extend this function to avoid having to repeat the `.text` when calling this function:
```suggestion
def find_element(parent, tag_name):
    tag = parent.find(f"{{*}}{tag_name}")
    if not tag:
        return None
    return tag.text
```